assessment.md

```diff
@@ -13,81 +13,512 @@
+  * N = None would indicate that someone did not use the issue tracker.
+  * I = Inadequate would indicate that an issue is posted but it is either
+  redundant or incomprehensible. Ex. ``Ajssjdjssd``(incomprehensible) or posting
+  that a program's main method is broken when three other people have done the
+  same thing.
+  * A = Adequate would indicate that the issue posted is neither redundant nor
+  incomprehensible, but isn't descriptive or clear. Ex. ``Your program is
+  broken.``
+  * G = Good would indicate that the issue posted is neither redundant nor
+  incomprehensible, and is moderately descriptive and clear. Ex. ``Your math
+  function is broken.``
+  * E = Excellent would indicate that the issue posted is neither redundant nor
+  incomprehensible, and is detailed and easy to understand and follow. Ex.
+  ``Your math function is broken because you forgot a semi-colon.`` Error
+  messages from the terminal can also be incorporated. Ex ``Terminal said that
+  the function was given too many arguments.``
 * Using the GitHub flow model
+  * N = None would indicate that someone pushed directly to the master branch
+  instead of pushing to one of the subbranches. Additionally, they had
+  nonsensical commit messages, and did not create or use repository branches or
+  forks correctly, and also performed incorrect merges and pull requests.
+  * I = Inadequate would indicate that forks, and repositories are made
+  correctly. Additionally, merges and pull requests are done correctly and the
+  commit messages can be understood, but the programmer pushed to the master
+  branch.
+  * G = Good would indicate that the issue posted is neither redundant nor
+  incomprehensible, and is moderately descriptive and clear.
+  * A = Adequate would indicate that the programmer correctly made subbranches
+  and did not push to the master branch. Merge and pull requests are done
+  correctly, commit messages can be understood. Additionally, all forks and
+  repositories were made correctly as well.
+  * E = An excellent branch would be a purposeful branch that has a descriptive
+  label. Had strong commit messages, in addition to using and creating
+  repository branches, forks, and pull request correctly.
 * Creating and using a repository branch
+  * N = Routinely committing edits to the master branch or not pushing to
+  correct branches receives a none grade
+  * I = Creating branches with unclear names and no descriptions will receive an
+  inadequate grade
+  * A = Branches named sensibly with unique code and descriptions that
+  adequately describe the branch's purpose receive an adequate grade
+  * G = Well documented changes to correct well-named and necessary branches
+  will receive a good grade
+  * E = Immaculately documented changes on branches that serve extremely
+  important purposes will receive an excellent grade
 * Creating and using a repository fork
+  * N = Pushing to the wrong fork or making forks that serve no purpose will
```

Comment:
As @Michionlion has stated previously, there should not be " will receive a __ grade." The indicators, such as "N = ," represents this grade.

Reply:
This has been changed. Thank you